web/styles/app_components.css

```diff
@@ -115,7 +115,6 @@ input::placeholder {
         box-shadow: none;
 
         cursor: pointer;
-        transition: all 0.2s ease;
```

Comment:
So this is on a broad button element; do you know why we had this transition declared?

Reply:
https://chat.zulip.org/#narrow/stream/9-issues/topic/theme.20transition